proceeding/tables/survey_table.tex

```diff
@@ -0,0 +1,23 @@
+%table
+
+\begin{table*}[h]
+
+\caption{Survey information.}
+\label{tab:a}
+\tabcolsep7pt\begin{tabular}{ || c|cc ||}
+\hline
+
+\textbf{Survey name} & \textbf{Telescope/Observatory} & \textbf{Frequency band}\\ \hline
+\textbf{Fermi color} & Fermi-LAT & \\
+\textbf{AKARI 90um} & AKARI & \\
+\textbf{CGPS-VGPS CONT} &  & \\
+\textbf{Spitzer GLIMPSE360} & Spitzer & \\
+\textbf{Haslam 408} &  & 408 MHz\\
+\textbf{IRIS Band 4-100um} & IRIS & \\
+\textbf{Planck R2 LFI Color 30-44-70 GHz} & Planck & 30-44-70 GHz\\
+\textbf{Planck R1 HFI Color 353-545-857 GHz} & Planck & 353-545-857 GHz\\
+\textbf{Planck R2 HFI Color 353-545-857 GHz} & Planck & 353-545-857 GHz\\
```

Comment:
Maybe combine the Planck ones and mention that there's several in the description?
This is anyways just a first selection, I think this will change a lot in the coming months.